frontend/sw360-portlet/src/main/resources/content/Language.properties

```diff
@@ -858,7 +858,7 @@ link.to.project=Link to Project
 link.to.projects=Link to Projects
 list.of.components.without.version.information=List of Components without version information
 list.of.invalid.packages.without.purl.or.name.or.version=List of invalid Packages without purl or name or version
-list.of.packages.without.vcs.information=List of Packages without VCS information
+list.of.packages.without.vcs.information=List of Packages with invalid or missing VCS information
```

Comment:
This a workaround until we migrate to React framework. Did not want to touch the jsp files involving liferay variables.